.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,39 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: 'Maven: Publish Packages to GitHub'
+
+on:
+  workflow_dispatch:
+# Release trigger does not work because the release job uses GITHUB_TOKEN
+#   release:
+#     types: [created]
+
+env:
+  JAVA_VERSION: '11'
+  JAVA_DISTRIBUTION: 'corretto'
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up JDK ${{ env.JAVA_VERSION }}
+        uses: actions/setup-java@v3
+        with:
+          java-version: ${{ env.JAVA_VERSION }}
+          distribution: ${{ env.JAVA_DISTRIBUTION }}
+          cache: 'maven'
+          server-id: github # Value of the distributionManagement/repository/id field of the pom.xml
+
+      - name: Build with Maven
+        run: mvn -B -e -ntp package
+
+      - name: Publish to GitHub Packages Apache Maven
+        run: mvn deploy -Pgithub-publish -DskipTests
```

Comment:
Going to need to add this profile to the pom

Reply:
Ah good catch. Added.